README.md

```diff
@@ -96,7 +97,22 @@ mvn orphedomos:multiplatform-build-deploy -Dorphedomos.build.architecture=linux/
   <targetArchitectures>linux/arm64</targetArchitectures>
   <targetArchitectures>linux/amd64</targetArchitectures>
 </targetArchitectures>
+
+
+```
```

Comment:
A (Administrative): here and in subsequent locations, I'd remove the new lines in these blocks. Let markdown format it and add spacing so it is consistent.

Reply:
Fixed.